src/Symfony/Component/Security/Core/Tests/Authorization/Voter/LegacyAbstractVoterTest.php

```diff
@@ -13,6 +13,16 @@
 
 use Symfony\Component\Security\Core\Authorization\Voter\AbstractVoter;
 
+class LegacyAbstractVoterTest extends AbstractVoterTest
```

Comment:
Maybe it's better to fix the tests in 2.7 and then merge it in 2.8 (where we'll add some scenarios because of deprecations). I'll open a PR soon for 2.7.